rootfs/usr/local/include/toolbox/kops/Makefile.defaults

```diff
@@ -10,13 +9,8 @@ export KOPS_STATE_STORE ?= s3://$(CLUSTER_STATE_BUCKET)
 
 export KOPS_CLOUD ?= aws
 export KOPS_ZONES ?= $(AWS_REGION)a
-
-export KOPS_MASTER_SIZE ?= t2.medium
 export KOPS_MASTER_ZONES ?= $(KOPS_ZONES)
 
-export KOPS_NODE_SIZE ?= t2.medium
```

Comment:
@aknysh I have redacted your comment as it referred to private repositories. Keep in mind that this is a public repo.